### PR TITLE
KEB with openstack

### PR DESCRIPTION
**Pull request by piotrmiskiewicz**
**Description**

Changes proposed in this pull request:
KEB works with openstack

**Related issue(s)**

### CODE REVIEW

File: components/kyma-environment-broker/internal/broker/plans.go [hunk 1]
```diff
@@ -68,6 +72,10 @@ func GCPRegions() []string {
 		"northamerica-northeast1", "southamerica-east1"}
 }
 
+func OpenStackRegions() []string {
+	return []string{"eu-nl-1", "eu-de-1"}
```
Comment (ebensom) [inline components/kyma-environment-broker/internal/broker/plans.go:76]: We will use eu-de-2 (Frankfurt) in EU, and ap-sa-1 for KSA

File: components/kyma-environment-broker/internal/provider/openstack_provider.go [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+type OpenStackInput struct {
+}
+
+func (p *OpenStackInput) Defaults() *gqlschema.ClusterConfigInput {
+	return &gqlschema.ClusterConfigInput{
+		GardenerConfig: &gqlschema.GardenerConfigInput{
+			DiskType:       ptr.String("pd-standard"),
+			VolumeSizeGb:   ptr.Integer(30),
+			MachineType:    "m1.large",
```
Comment (ebensom) [inline components/kyma-environment-broker/internal/provider/openstack_provider.go:24]: To match Azure, GCP and AS production plans, we could use m2.xlarge (8 CPU / 16 GB RAM) or m1.2xlarge (8 CPU / 32 GB RAM). But on CCloud I'm a bit worried about node unavailability more than with other providers

File: components/kyma-environment-broker/internal/provider/openstack_provider.go [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+			Region:         DefaultOpenStackRegion,
+			Provider:       "openstack",
+			WorkerCidr:     "10.250.0.0/19",
+			AutoScalerMin:  3,
```
Comment (ebensom) [inline components/kyma-environment-broker/internal/provider/openstack_provider.go:28]: To match the bigger VM size, this would be reduced to 2.

**Review by polskikiel — APPROVED** [review]
lgtm

**Review by polskikiel — APPROVED** [review]

**Pull request merged**